Source/DataSources/VelocityVectorProperty.js

```diff
@@ -0,0 +1,184 @@
+/*global define*/
+define([
+    '../Core/Cartesian3',
+    '../Core/defaultValue',
+    '../Core/defined',
+    '../Core/defineProperties',
+    '../Core/DeveloperError',
+    '../Core/Event',
+    '../Core/JulianDate',
+    './Property'
+], function(
+    Cartesian3,
+    defaultValue,
+    defined,
+    defineProperties,
+    DeveloperError,
+    Event,
+    JulianDate,
+    Property) {
+    'use strict';
+
+    /**
+     * A {@link Property} which evaluates to a {@link Cartesian3} vector
+     * based on the velocity of the provided {@link PositionProperty}.
+     *
+     * @alias VelocityVectorProperty
+     * @constructor
+     *
+     * @param {Property} [position] The position property used to compute the velocity.
+     *
+     * @example
+     * //Create an entity with position and orientation.
```

Comment:
The comment is still incorrect.  Change it to something like `//Create an entity with a billboard rotated to match its velocity`